### ISSUE TITLE
Add ngModel in video id 

### ISSUE THREAD

**Issue by RamyHakam** [sylc/ng2-youtube-player-mini#2]
How I can Detect the Video ID from user input using ngModel 
As well as changing the url will reload the video with the new one ?

**Comment by arLevi** [comment]
Yes, I'm having the same problem, after changing the `videoId` in the element - nothing happens to the video itself

**Comment by kyungjoongo** [comment]
same problem ..How to change video?
